vuvuzela_test.go

```diff
@@ -0,0 +1,304 @@
+package vuvuzela
+
+import (
+	"crypto/rand"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"vuvuzela.io/vuvuzela/coordinator"
```

Comment:
Keep the `vuvuzela.io` imports together at the end of the import list.

Reply:
Oops, amended